_posts/2018-11-11-fs.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: The Choices We Make
+---
+
+The common theme behind all the container runtime alternatives--Kata, Gvisor and Nabla--is to improve container isolation.
+However, to achieve the same goal, each solution adopts a different approach. 
+Kata employs the virtual machine isolation, Gvisor the xxx sandbox [TODO], and Nabla the libOS+Seccomp based attack surface minimization.
```

Comment:
Kata employs virtual machine isolation, while Gvisor and Nabla use libOS+Seccomp based attack surface minimization.